src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationFactory.java

```diff
@@ -577,7 +579,11 @@ public DependencyGraph aggregateDirect(List<DetectCodeLocation> detectCodeLocati
     }
 
     public DependencyGraph aggregateTransitive(List<DetectCodeLocation> detectCodeLocations) throws DetectUserFriendlyException {
-        return auditLog.namedPublic("Transitive Aggregate", "TransitiveAggregate", () -> new AggregateModeTransitiveOperation(new SimpleBdioFactory()).aggregateCodeLocations(directoryManager.getSourceDirectory(), detectCodeLocations));
+        return auditLog.namedPublic("Transitive Aggregate", "TransitiveAggregate", () -> new AggregateModeTransitiveOperation(new FullAggregateGraphCreator(new SimpleBdioFactory())).aggregateCodeLocations(directoryManager.getSourceDirectory(), detectCodeLocations));
+    }
+
+    public DependencyGraph aggregateSubProject(List<DetectCodeLocation> detectCodeLocations) throws DetectUserFriendlyException {
+        return auditLog.namedPublic("SubProject Aggregate", () -> new AggregateModeSubProjectOperation(new FullAggregateGraphCreator(new SimpleBdioFactory())).aggregateCodeLocations(directoryManager.getSourceDirectory(), detectCodeLocations));
```

Comment:
```suggestion
        return auditLog.namedPublic("SubProject Aggregate", "SubprojectAggregate", () -> new AggregateModeSubProjectOperation(new FullAggregateGraphCreator(new SimpleBdioFactory())).aggregateCodeLocations(directoryManager.getSourceDirectory(), detectCodeLocations));
```

Can we add the phone home tag here so we know how many people are using it? It be cool to get a feature with decent analytics from the get-go￼.

Reply:
Done.